packages/aws/data_stream/apigateway_metrics/manifest.yml

```
@@ -0,0 +1,32 @@
+title: AWS API Gateway metrics
+type: metrics
+streams:
+  - input: aws/metrics
+    vars:
+      - name: period
+        type: text
+        title: Collection Period
+        multi: false
+        required: true
+        show_user: true
+        default: 5m
+      - name: data_granularity
+        type: text
+        title: Data Granularity
+        multi: false
+        required: false
+        show_user: false
+      - name: regions
+        type: text
+        title: Regions
+        multi: true
+        required: false
+        show_user: true
+      - name: latency
```

Stealing the `latency` description from https://github.com/elastic/integrations/pull/6231 

```suggestion
      - name: latency
        description: To address latency issues between certain AWS services and CloudWatch, specify a latency parameter to adjust the collection start time and end time in Metricbeat such as `5m`.
```